Assignment2/sebufer-SergioBujanFernandez2.json

```diff
@@ -0,0 +1,31 @@
+"@context":{
+		"zar":"https://www.w3.org/api/general",
+		"rdfs":"https://www.w3.org/api/general#",
+		"name":"rdfs:label",
+		"hometown": "rdfs:label",
+		"age": "rdfs:integer",
+		"alma_mater": "rdfs:label",
+		"parents": "rdfs:label",
+		"subjects": "rdfs:subjects",
+		"name": "rdfs:label",
+		"teachers": "rdfs:label",
+		"description": "rdfs:label",
+}
+
+"@id": "https://www.w3.org/api/info"
+"@types": "name": "Sergio Bujan Fernandez",

```

Comment:
@types doesn't exist, the correct is "@type" and you should use to specify a type of object, in this case you should define that you are a Person, Student, or any type that you chose. Then you define in other line your name.